app/controllers/devise/paranoid_verification_code_controller.rb

```diff
@@ -41,4 +41,10 @@ def authenticate_scope!
     send(:"authenticate_#{resource_name}!")
     self.resource = send("current_#{resource_name}")
   end
+
+  def redirect_route
+    Devise.paranoid_verification_code_redirect_location ||
+      stored_location_for(scope) ||
+      :root
+  end
```

Comment:
Perhaps we want to respect the stored location first and then fall back to the configured default.

```suggestion
  # Where to redirect to follow successful verification of the code
  # @return [String, Symbol]
  def redirect_route
    stored_location_for(scope) || Devise.paranoid_verification_code_redirect_location
  end
```

Reply:
The idea behind this setting was that it was an override setting, not a configured default setting. That's also why I defined it as `nil`, so that if it wasn't set it would just use the pre-existing logic.

Reply:
My suggestion should be equivalent except that it pulls the definition for what the default is into the configuration code instead of leaving it buried in the controllers.

Reply:
The biggest difference here is that the configuration you propose would completely ignore any 'stored location'. I don't think that's what we want, however.

Reply:
I think it is - the stored location is always set by the controller, so if we put it first on the priority list then the new config will never be used.

See https://github.com/devise-security/devise-security/blob/2b8a56884125dae487f1368cba42fec7e78b919a/lib/devise-security/controllers/helpers.rb#L64